Resources/ServerInfo/Rules.txt

```diff
@@ -57,18 +62,19 @@
 	• Fights and confrontations is completely disallowed on the safezone such as shooting, attacking or any forms of harm.
 	• Pirate actions may not begin on the Frontier. This includes stowing away while at the Frontier with intentions to steal the ship, outright stealing the ship or its cargo from the dock, selling someone else’s ship you hadn’t pirated outside the safe zone, or joining the crew under false pretenses with the intention to mutiny and take the ship.
 	• Dangerous research like artifacts and anomalies must not be done while inside the safezone boundaries.
-	• Do not transport any kind of dangerous creature inside the station.
+	• Do not transport any kind of creature, contraband or objects that might cause harm/death or bait into a conflict inside of the frontier station.
```

Comment:
Contraband has not been defined within this document.  This'll be the first thing most people skim through - if you leave it at "objects", you leave out the question of "what exactly is contraband".  The player that finds contraband without knowing what it is only to show up on station didn't do anything malicious.

I think a rewrite into two points is easier to understand.  Does this seem like what you were trying to say?

```suggestion
	• Do not transport creatures that can cause harm/death inside of the Frontier.
	• Do not bring objects onto the Frontier with intent to cause harm/death, or bait others into a conflict.
```

Reply:
the second one sounds redundant but i do think removing the contraband part is fine, im just going to reduce it to creatures and objects and that will cover contraband, and even artifacts or radiation objects.

Reply:
This is mostly being done for the problem people that abuse that security needs to arrest the contraband yet they go inside the station looking like a contraband piñata.

Reply:
with recent changes with the trading post cargo on the station will be chopped off eventually, about intention whoops yeah let me add that because there is cases where people bring roadkill mobs that where on space too lol